paymentsource/client_test.go

```diff
@@ -154,54 +274,58 @@ func TestSourceUpdate(t *testing.T) {
 	customer.Del(cust.ID)
 }
 
-func TestSourceVerify(t *testing.T) {
-	bankAccountParams := &stripe.BankAccountParams{
-		Country:  "US",
-		Currency: "usd",
-		Routing:  "110000000",
-		Account:  "000123456789",
-	}
-	tokenParams := &stripe.TokenParams{
-		Bank: bankAccountParams,
+func TestSourceBankAccountVerify(t *testing.T) {
+	baTok, err := token.New(&stripe.TokenParams{
+		Bank: &stripe.BankAccountParams{
+			Country: "US",
+			Currency: "usd",
+			Routing: "110000000",
+			Account: "000123456789",
+			Name: "Jane Austen",
+			AccountHolderType: "individual",
+		},
+	})
+
+	if baTok.Bank.Name != "Jane Austen" {
+		t.Errorf("Bank account name %q was not Jane Austen as expected.", source.BankAccount.Name)
 	}
-	token, err := token.New(tokenParams)
 
 	if err != nil {
 		t.Error(err)
-		return
 	}
 
 	customerParams := &stripe.CustomerParams{}
-	customerParams.SetSource(token.ID)
-
-	cust, err := customer.New(customerParams)
-
+	customer, err := customer.New(customerParams)
 	if err != nil {
 		t.Error(err)
-		return
 	}
 
-	amounts := [2]uint8{32, 45}
+	source, err := New(&stripe.CustomerSourceParams{
+		Customer: customer.ID,
+		Source: &stripe.SourceParams{
+			Token: baTok.ID,
+		},
+	})
+
+	amounts := [2]uint8{32, 45};
 
 	verifyParams := &stripe.SourceVerifyParams{
-		Customer: cust.ID,
-		Amounts:  amounts,
+		Customer: customer.ID,
+		Amounts: amounts,
 	}
 
-	source, err := Verify(cust.DefaultSource.ID, verifyParams)
+	src, err := Verify(source.ID, verifyParams)
 
 	if err != nil {
 		t.Error(err)
 		return
 	}
 
-	target := source.BankAccount
+	target := src.BankAccount
 
 	if target.Status != bankaccount.VerifiedAccount {
 		t.Errorf("Status (%q) does not match expected (%q) ", target.Status, bankaccount.VerifiedAccount)
 	}
-
-	customer.Del(cust.ID)
```

Comment:
@wbronitsky Any reason to pull this `customer.Del` out? It's a little crazy, but this is actually how this binding's test suite cleans itself up (or tries to anyway).